### DISCUSSION TITLE
RtpMic for the A11 rom on ThinkSmart View?

### DISCUSSION THREAD

**Discussion by mngarchow** [dinki/View-Assist#8] [Q&A] [answered]
Hi all, 

Now that I have Android11 on the ThinkSmart view, I am trying to follow the rest of the wiki to the full View-Assist experience working. I would like to use RtpMic, but I can't find it on the google play store. Is it possible to load it on Android 11? Is anyone able to help me with it? Thanks!

**Comment by dinki** [comment]
I was going to point you to the wiki page and just realized that I only put the link on the Youtube video.  I'll edit it but you can use this link from your computer and send it over to the ThinkSmart:

https://play.google.com/store/apps/details?id=com.rtpmic&hl=en_US&gl=US

Know that the wiki is still a work in progress but I'm chipping away getting it populated.  Things posted now will work but the improvements will be worth the wait.

**Reply by mngarchow** [reply]
Oh the link is on the wiki on the page "Android Software". I just tried installing it from my computer and while the app is available for my phone, it is greyed out for the ThinkSmart. In the wiki (also on the Android Software page) you mention installing it on a Fire Tablet, but have you installed rtpmic on the ThinkSmart with Android 11? I confirmed that I am able to install IP Webcam, I just thought rtpmic would be better.

**Reply by dinki** [reply]
https://youtu.be/SkyErx7TE30?si=hDDCgR6CAFnHyzym&t=17

Very odd.  If you click on my video above you'll see that I use the Playstore to search and find RTPMic and install without issue.  All videos on the channel, with the exception of the very first one for the contest demo, are using the ThinkSmart device.

I am really unsure why you are not able to find the application with a search.  I am assuming you followed my previous install videos for the Thinksmart, are using the same ROM, and have authenticated your account with Google Playstore.

I do find the experience with RTPMic to be better than IPWebcam in that I THINK it uses less resources.  Both will work fine though.

**Reply by mngarchow** [reply]
> I am really unsure why you are not able to find the application with a search. I am assuming you followed my previous install videos for the Thinksmart, are using the same ROM, and have authenticated your account with Google Playstore.

I did follow your videos and I used the A11 link that you added in the description after you recorded the video. I checked and the google play store shows my account as being logged in. On the ThinkSmart, I can change the device from "this device" to one of my other tablets and RtpMic does show up in the search list. I am just not able to download it "This phone is not compatible with this app". I'm not sure what about my installation was different from yours that led to this. Oh well, I will just use IPWebcam. Thanks for your time.

**Reply by Kilthor** [reply, marked as answer]
I had same problem getting RTPmic to install on the thinksmart A11
Said this version of RTPmic was not supported
I used a third-party installer to install it. APKPure I think was the name of it

**Reply by mngarchow** [reply]
After comparing the two again, I decided to sideload RTPMic via a 3rd-party website. I hope the website I got it from is legitimate.

**Comment by dinki** [comment]
Apologies to you all.  I have no idea why I can see and install the app without issue and you folks cannot.  I will say that I find running the RTPMic app and Wallpanel seem to give a much more stable platform than using IPWebcam.  I am certainly open to testing other software you folks might find too.  Maybe someday we'll have wakeword on Android and we can eliminate Stream Assist altogether.

**Reply by janstadt** [reply]
I loaded the latest Lineage 15 rom on my thinksmart and am running rtpmic and wallpanel but it doesnt seem to really work super well. Stream Assist goes to wake, but then just sits there and if i go into my debug view of the assist it looks like openwakeword just sits there and spins. Anyone else seeing this?

**Reply by dinki** [reply]
I'm not 100% but I know that @adhdsquir put in a PR that fixed issues with HA 2024.5 and another fix.  I downloaded those changes separately and applied.  I know that this is not part of the standard release but asked AlexxIT (SA) to do a point release and he said he will in a few days.  You can try downloading the changed files and replacing to see if that fixes things for you.  Hope that helps.  

Also, you can pop into Discord and folks there may be able to help troubleshoot further.
